local-development/common.sh

```diff
@@ -11,5 +11,13 @@ function fix_linux_internal_host() {
   fi
 }
 
-fix_linux_internal_host
-vendor/bin/heroku-php-apache2 -F fpm_custom.conf -i local_php.ini workbench
\ No newline at end of file
+if [ -d .profile.d ]; then
```

Comment:
This mimics more closely what Heroku does (https://devcenter.heroku.com/articles/buildpack-api#profile-d-scripts)